src/blocks/blocks/form/style.scss

```diff
@@ -6,8 +6,8 @@
 	--labelColor: inherit;
 	--inputWidth: 100%;
 	--requiredColor: red;
-	--submitBackground: var(--wp-admin-theme-color);
-	--submitBackgroundHover: var(--wp-admin-theme-color-darker-10);
+	--submitBackground: inherit;
+	--submitBackgroundHover: inherit;
 	--submitColor: white;
```

Comment:
@HardeepAsrani, @arinaturcu any idea on how I can set those variables to inherit the theme values (like the ones from `wp-block-button__link`).

Reply:
They weren't working because --wo-admin-theme doesn't load on front. @Soare-Robert-Daniel

Reply:
It is not about that. Is about inheriting the colors from the `wp-blocks-button__link`. With CSS vars I did not find a solution, so I change it and made it as we did it in the past: https://github.com/Codeinwp/otter-blocks/pull/797/commits/68b3988121547bc3eea26361667b5634fa1ab2d7.